source/configuration_manual/authentication/auth_policy.rst

```diff
@@ -168,6 +168,46 @@ produces
 
 for this to work.
 
+List of fields
+--------------
+
+All fields supported by :ref:variables-auth can be used. In addition, you can use following fields:
+
+``hashed_password``
+  User's password hashed with password hash algorithm.
+
+``requested_username``
+  Username that was requested. This is normalization field which contains the correct username,
+  depending on wheter this is master user login or not.
+
+``fail_type``
+  .. versionadded:: v3.0.0;v2.4.0
+
+  Will be filled with the reason request failed. This can take one of
```

Comment:
* "Will be filled with" seems unnecessary. Just start with "The reason .."
 * the request